package.json

```diff
@@ -2,12 +2,13 @@
   "name": "mini-typescript",
   "version": "1.0.0",
   "description": "A miniature model of the Typescript compiler",
-  "main": "index.js",
+  "main": "compiler/index.js",
   "scripts": {
     "build": "tsc",
     "test": "rm baselines/local/*; tsc && node test.js",
     "accept": "mv baselines/local/* baselines/reference/",
-    "run": "node index.js"
+    "run": "node compiler/index.js",
+    "mtsc": "node compiler/index.js"
   },
```

Comment:
`npm run run` haha, so I gave it a different name

Reply:
Might as well delete `run` then, I copied that early on from the F# version and never deleted it.